### PR TITLE
Upath ubi 8.4

### PR DESCRIPTION
**Pull request by vikasgupta8**
Have set default version to 1.2.0 and can supply other versions through command line.
Tested with v1.2.0 and v2.0.1

### CODE REVIEW

File: u/upath/upath_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : upath
+# Version       : v1.2.0, v2.0.1
+# Source repo   : https://github.com/anodynos/upath
+# Tested on     : RHEL ubi 8.4
```
Comment (amitsadaphule) [inline u/upath/upath_ubi_8.4.sh:6]: Does this mean that the script has been validated on RHEL 8.4 as well as UBI 8.4? The scripts which work on UBI 8.4 almost certainly work on RHEL 8.4, but the reverse is not always true.

File: u/upath/upath_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        exit 0
+fi
+
+cd $HOME_DIR/$PACKAGE_NAME/
+
+git checkout $PACKAGE_VERSION
+PACKAGE_VERSION=$(jq -r ".version" package.json)
+
+npm install -g grunt && npm install -g grunt-cli
+cd $HOME_DIR/$PACKAGE_NAME/
```
Comment (amitsadaphule) [inline u/upath/upath_ubi_8.4.sh:44]: We're already inside `$HOME_DIR/$PACKAGE_NAME/` as per line 38.

File: u/upath/upath_ubi_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME" > /home/tester/output/clone_fails
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails" > /home/tester/output/version_tracker
+        exit 0
+fi
+
+cd $HOME_DIR/$PACKAGE_NAME/
+
+git checkout $PACKAGE_VERSION
+PACKAGE_VERSION=$(jq -r ".version" package.json)
```
Comment (amitsadaphule) [inline u/upath/upath_ubi_8.4.sh:41]: Why are we setting `PACKAGE_VERSION` again here? Does the tag name does not match with the version from `package.json`?

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: u/upath/upath_ubi_8.4.sh (hunk 1 above)
Comment (vikasgupta8) [reply u/upath/upath_ubi_8.4.sh]: It was tested on ubi 8.4. I have updated the script.

File: u/upath/upath_ubi_8.4.sh (hunk 2 above)
Comment (vikasgupta8) [reply u/upath/upath_ubi_8.4.sh]: Yes removed the command.

File: u/upath/upath_ubi_8.4.sh (hunk 3 above)
Comment (vikasgupta8) [reply u/upath/upath_ubi_8.4.sh]: I have removed the re-setting of PACKAGE_VERSION

**Review by amitsadaphule — CHANGES_REQUESTED** [review]
Please resolve the conflicts on the PR.

**Comment by vikasgupta8** [comment]
Resolved conflicts.

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Pull request merged**